updatecli/updatecli.d/charts/certmanager.yaml

```diff
@@ -1,50 +0,0 @@
-title: Bump cert-manager helm chart version
```

Comment:
What is the reason for removing the tracking for cert-manager ?

Reply:
A bad merge, I've renamed the file `cert-manager.yaml` but failed to properly include it in this PR. (Cf previous one)